### ISSUE TITLE
Chrome: Lazy Load with data-thumb causes problem with first load

### ISSUE THREAD

**Issue by devFinar** [artpolikarpov/fotorama#551]
If we use Lazy Load with data-thumb, we have code like this:

```
<script src='https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js'></script>
<link  href='https://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.css' rel='stylesheet'>
<script src='https://cdnjs.cloudflare.com/ajax/libs/fotorama/4.6.4/fotorama.js'></script>

<div class='fotorama' data-allowfullscreen='native' data-nav='thumbs'>
  <a href="1.jpg" data-thumb="1_thumb.jpg"></a>
  <a href="2.jpg" data-thumb="2_thumb.jpg"></a>
  <a href="3.jpg" data-thumb="3_thumb.jpg"></a>
</div>  
```

If the page is opened in Google Chrome, first time the gallery looks broken: only the first thumb is shown with no original picture. 

If we press F5 it works OK. 

If we press CTRL+F5 the problem comes again.

**Comment by artpolikarpov** [comment]
Can’t reproduce. Do you see this bug on [this page](https://fotorama.io/docs/4/shuffle/)? If yes, please reopen the issue.
